scripts/stability.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+const fs = require('fs');
+const packageJsonPath = process.argv[2] || path.join(process.cwd(), 'package.json');
```

Comment:
Can we space up between `import` and actual constant definitions?

Additionally, this feels kinda subject to incorrect usage... Since I could pass 300 arguments and it'd happily silently ignore 299 of them.

Reply:
Done